installer/pyinstaller/build-linux.sh

```diff
@@ -21,9 +21,16 @@ else
     is_nightly="false"
 fi
 
+if [ "$CI_OVERRIDE" = "1" ]; then
+  build_folder="aws-sam-cli-dev"
+  build_binary_name="sam-dev"
```

Comment:
Maybe let's give it a different name to not confuse with the samdev command? I know we can name whichever binary name but just an idea for ourselves to reference easier

Reply:
`sam-ci-build` ? (Open to others too). Thought of `gamma` as well.

Reply:
Updated with `sam-beta` as discussed offline